Directory parsing when running tests

Reporting an issue with running ``make test``.  When I clone the repository into a directory with the same name as the repository (e.g. ``~/git/modelica-json/modelica-json`` or ``~/git/modelica-json-2/modelica-json``, then ``make test`` fails with:

```
:~/git/modelica-json-2/modelica-json$ make test
npm test

> cdl_export_english@0.0.1 test /home/dhbubu18/git/modelica-json-2/modelica-json
> mocha --timeout 120000 && standard



  jsonquery.js
    testing classDefinition
      ✔ testing structure
    testing classSpecifier
      ✔ checking for empty class_specifier -> should return an error
      ✔ checking with long_class_specifier
      ✔ checking with short_class_specifier
      ✔ checking with der_class_specifier
    testing longClassSpecifier
      ✔ testing structure
      ✔ testing missing identifier
      ✔ testing missing composition
    testing composition
      ✔ testing structure
      ✔ testing missing element_list
    testing classModification
      ✔ testing structure
      ✔ testing empty arguments
    testing elementList
      ✔ testing structure
      ✔ testing constraining_clause without replaceable
    testing element_sections
      ✔ testing structure
    testing external_function_call
      ✔ testing structure
    testing external_composition
      ✔ testing structure
    testing component_clause
      ✔ testing structure
    testing extends_clause
      ✔ testing structure
    testing import_clause
      ✔ testing structure
    testing component_list
      ✔ testing structure
    testing declaration
      ✔ testing structure
    testing element_modification
      ✔ testing structure: element_modification
      ✔ testing structure: graphicAnnotationObj
    testing element_modification_or_replaceable
      ✔ testing structure
    testing element_redeclaration
      ✔ testing structure
    testing epxression_list
      ✔ testing structure

  parser.js
    Testing parse from Modelica to raw Json, in "cdl" parsing mode
      1) Testing unmodified json for equality, "cdl" mode
    Testing parse from Modelica to raw Json, in "modelica" parsing mode
      ✔ Testing unmodified json for equality, "modelica" mode (11586ms)
    Testing parse from Modelica to Json, in "cdl" parsing mode
      2) Testing json for equality, "cdl" mode
    Testing parse from Modelica to Json, in "modelica" parsing mode
      ✔ Testing json for equality, "modelica" mode (26114ms)


  29 passing (38s)
  2 failing

  1) parser.js
       Testing parse from Modelica to raw Json, in "cdl" parsing mode
         Testing unmodified json for equality, "cdl" mode:
     Error: ENOENT: no such file or directory, open '/home/dhbubu18/git/modelica-json-2/modelica-json/raw-json/test/FromModelica/Block1.json'
      at Object.openSync (fs.js:498:3)
      at Object.readFileSync (fs.js:394:35)
      at /home/dhbubu18/git/modelica-json-2/modelica-json/test/test_parser.js:71:35
      at Array.map (<anonymous>)
      at Context.<anonymous> (test/test_parser.js:54:17)
      at processImmediate (internal/timers.js:464:21)

  2) parser.js
       Testing parse from Modelica to Json, in "cdl" parsing mode
         Testing json for equality, "cdl" mode:
     Error: ENOENT: no such file or directory, open '/home/dhbubu18/git/modelica-json-2/modelica-json/json/test/FromModelica/Block1.json'
      at Object.openSync (fs.js:498:3)
      at Object.readFileSync (fs.js:394:35)
      at /home/dhbubu18/git/modelica-json-2/modelica-json/test/test_parser.js:71:35
      at Array.map (<anonymous>)
      at Context.<anonymous> (test/test_parser.js:54:17)
      at processImmediate (internal/timers.js:464:21)



npm ERR! Test failed.  See above for more details.
Makefile:33: recipe for target 'test' failed
make: *** [test] Error 1
```

And I notice for instance:

```
~/git/modelica-json-2/modelica-json/raw-json$ ls
2
```

Which indicates there's an issue with determining where to create the files in the associated failed tests if the containing directory of the repo also contains ``modelica-json``.

--- a/lib/util.js
+++ b/lib/util.js
@@ -133,8 +133,7 @@ function relativePath (filePath) {
   var paths = getMODELICAPATH()
   var bases = []
   paths.forEach(ele => {
-    var temp = path.parse(ele).base
-    if (temp.length > 0) bases.push(temp)
+    if (ele.length > 0) bases.push(ele)
   })
   var relPa = filePath
   for (var i = 0; i < bases.length; i++) {

